accelerator/config.c

```diff
@@ -304,7 +303,7 @@ status_t ta_config_set(ta_cache_t* const cache, iota_client_service_t* const ser
   }
 
   if (iota_client_core_init(service)) {
-    ta_log_critical("Initializing IRI connection failed!\n");
+    ta_log_error("Initializing IRI connection failed!\n");
```

Comment:
We forgot to pass `CONFIG_LOGGER` here and elsewhere in this file, please modify it :smile:

Reply:
Sorry, I don't get it. Why and where should I pass CONFIG_LOGGER?